.make.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cmake . -B ./build && cmake --build ./build -j 4
```

Comment:
```suggestion
cmake . -B ./build && cmake --build ./build -j $(nproc --all)
```
This would allow to use 100% of processing units on all types of computers.